x-pack/plugins/security_solution/public/common/containers/matrix_histogram/use_matrix_histogram_async.ts

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Created this version because I needed a hook that did not immediately fire, but instead provided a "start" method that could be invoked when needed.